include/tera/table_descriptor.h

```diff
@@ -1,190 +1,214 @@
 // Copyright (c) 2015, Baidu.com, Inc. All Rights Reserved
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+//
+// TableDescriptor/LocalityGroupDescriptor/ColumnFamilyDescriptor are used to
+// describe the schema of a table when creating a table.
+// Should not use these classes if nessesary. User can also describe a table
```

Comment:
It's not a nessesary to use these classes. User is encouraged to describe a table ...